components/Filters/ActionsFilterToolbar.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { c } from 'ttag';
+import { Button, PrimaryButton, useModals } from 'react-components';
+
+import AddFilterModal from '../../containers/Filters/AddFilterModal';
+
+function ActionsFilterToolbar() {
+    const { createModal } = useModals();
+
+    const handleClickAdd = (type) => () => {
+        createModal(<AddFilterModal type={type} />);
+    };
+
+    return (
+        <>
+            <PrimaryButton onClick={handleClickAdd()}>{c('Action').t`Add Filter`}</PrimaryButton>
+            <Button onClick={handleClickAdd('complex')} className="ml1">
+                {c('Action').t`Add Sieve Filter`}
```

Comment:
Don't hover capitalize text please. I know on Zeplin it's the case, but it's a mistake ...